test/e2e/s3/cross-account.sh

```diff
@@ -8,12 +8,18 @@ source "$SCRIPTS_DIR/lib/common.sh"
 source "$SCRIPTS_DIR/lib/k8s.sh"
 source "$SCRIPTS_DIR/lib/testutil.sh"
 
-
 # AWS_ACCOUNT_ID_ALT should be configured to allow cross account resources management
 # from AWS_ACCOUNT_ID
 AWS_ACCOUNT_ID_ALT=${AWS_ACCOUNT_ID_ALT:-""}
 AWS_PROFILE_ALT=${AWS_PROFILE_ALT:-""}
 AWS_REGION_ALT="eu-west-2"
+TESTING_NAMESPACE=${TESTING_NAMESPACE:-"testing"}
+ASSUME_POLICY_ARN=${RECOMMENDED_ROLE_ARN:="s3FullAccess"}
```

Comment:
Where does RECOMMENDED_ROLE_ARN come from?

Reply:
I was thinking about reading this role from `$service/config/iam/recommended-policy-arn` file, but i don't think it's correct anymore. Changing to `ASSUME_POLICY_ARN=${ASSUME_POLICY_ARN:="s3FullAccess"}`